tests/t401-qfunction-f.f

```diff
@@ -0,0 +1,126 @@
+c-----------------------------------------------------------------------
```

Comment:
This is the first time we have tested that the Fortran ctx works. This does not currently come up in the Nek5k PR.